src/js/index.js

```diff
@@ -23,6 +23,19 @@ window.transfers = transfers
 window.urlParams = urlParams
 window.utils = utils
 
+if (process.env.nearNetworkId === 'testnet' &&
+    process.env.ethNetworkId === 'ropsten') {
+  window.bridgeName = 'Near Testnet - Ropsten'
+} else if (process.env.nearNetworkId === 'testnet' &&
+    process.env.ethNetworkId === 'rinkeby') {
+  window.bridgeName = 'Near Testnet - Rinkeby'
+} else if (process.env.nearNetworkId === 'mainnet' &&
+    process.env.ethNetworkId === 'main') {
+  window.bridgeName = 'Near - Ethereum'
+} else {
+  window.bridgeName = 'Unknown'
+}
```

Comment:
nit: this is a little tough to read. Maybe clean it up with a switch statement:

```js
switch (`${process.env.nearNetworkId}.${process.env.ethNetworkId}`) {
  case 'testnet.ropsten': window.bridgeName = 'Near Testnet - Ropsten'; break
  case 'testnet.rinkeby': window.bridgeName = 'Near Testnet - Rinkeby'; break
  case 'mainnet.main': window.bridgeName = 'Near - Ethereum'; break
  default: window.bridgeName = 'Unknown'
}
```

Or an object:

```js
window.bridgeName = {
  testnet_ropsten:  'Near Testnet - Ropsten',
  testnet_rinkeby:  'Near Testnet - Rinkeby',
  mainnet_main:  'Near - Ethereum',
}[`${process.env.nearNetworkId}_${process.env.ethNetworkId}`] || 'Unknown'
```

Reply:
agree, thanks !